src/ripple/app/tests/common_ledger.cpp

```diff
@@ -112,12 +112,11 @@ STTx
 parseTransaction(TestAccount& account, Json::Value const& tx_json, bool sign)
 {
     STParsedJSONObject parsed("tx_json", tx_json);
-    std::unique_ptr<STObject> sopTrans = std::move(parsed.object);
-    if (sopTrans == nullptr)
+    if (!parsed.object)
         throw std::runtime_error(
-        "sopTrans == nullptr");
-    sopTrans->setFieldVL(sfSigningPubKey, account.pk.getAccountPublic());
-    auto tx = STTx(*sopTrans);
+        "object not parseable");
```

Comment:
"very little" is subjective. More accurate would be to say "less." It is true that vertical information density is reduced but in return you get less horizontal variance. Vertical information density is not the sole factor in code readability.